doc/sphinx/language/core/sections.rst

```diff
@@ -131,6 +131,90 @@ usable outside the section as shown in this :ref:`example <section_local_declara
    Notice the difference between the value of :g:`x'` and :g:`x''` inside section
    :g:`s1` and outside.
 
+.. _visibility-attributes-sections:
+
+Summary of locality attributes in a section
+-------------------------------------------
+
+The following table sums up the locality of vernacular commands in sections, when
+outside the section where it was entered.
+A similar table for :cmd:`Module` can be found
```

Comment:
Perhaps you could include an asterisk on entries where the module and section behavior is different (in both tables)?  Won't users frequently want to compare the behavior of the two?

(I would have suggested making a single table that covers both sections and modules, but that likely won't fit the page width and may be hard to read.)

Reply:
I think they are very different (and locality attributes are less available in sections), would you have suggestions?

Reply:
Now that I can see both tables, the section table data appears fairly simple/relatively consistent, so I wouldn't bother with asterisks.